### PR TITLE
Add opportunity to trade jump drives for headpats

### PR DESCRIPTION
**Pull request by ziproot**
Wait for @AvianGeneticist approval as he made the initial conversation.

### CODE REVIEW

**Comment by AvianGeneticist** [comment]
Should probably have some indication that the "it's just four words" line is spoken by the player. Otherwise, I like it.

**Review by TheGiraffe3 — APPROVED** [review]
I mean, the grammar/writing looks good... but why?

**Comment by AvianGeneticist** [comment]
*clears throat*
Hehehehehe headpat

File: data/side missions.txt [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Based on "Unfettered Jump Drive 4" by Michael Zahniser.
+# Initial conversation and idea by AvianGeneticist: https://discord.com/channels/251118043411775489/1031714960528646194/1148660182310723624.
+# Combined mission by ziproot.
+disable mission "Unfettered Jump Drive 4"
+mission "Unfettered Jump Drive 4b"
+	job
+	repeat
+	name "Sell Jump Drive"
+	description "Exchange a jump drive on your flagship or in cargo for money, permission to pet the Unfettered, or for weapons, if an outfitter is available."
```
Comment (Hurleveur) [inline data/side missions.txt:9]: Isnt this too long of a description?

File: data/side missions.txt [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+					goto money
+				`	(Ask to pet the Unfettered.)`
+					goto headpat
+			action
+				outfit "Tripulse Shredder" 2
+			`	The two Shredders are loaded onto your ship, in exchange for your jump drive.`
+				goto trade
+
+			label headpat
+			`<tab>"You want this jump drive? Oh, honey, you know the fee," you say.`
```
Comment (Hurleveur) [inline data/side missions.txt]: Honey's pushing it a bit lmao

File: data/side missions.txt [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+				to display
+					has "jump drives for headpats"
+			`<tab>"You want this jump drive? Give me permission to pet you," you say.`
+				to display
+					not "jump drives for headpats"
+			`	"..."`
```
Comment (Hurleveur) [inline data/side missions.txt:61]: Just an idea, but what if they have a 5% chance to get mad or smth like that?

**Review by Hurleveur — COMMENTED** [review]
Sorry it took so long for me to look at it, I dont get notificztions for PR opened to my repos for some reason lol

File: data/side missions.txt (hunk 1 above)
Comment (ziproot) [reply data/side missions.txt:9]: No

**Comment by Hurleveur** [comment]
I love that machette but I'd change their dialog first to say they're not so agreeing this time

**Comment by ziproot** [comment]
OK, I changed it so that you have a chance to abort

File: data/side missions.txt [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+			branch nochange
+				random < 50 + "unfettered headpat attitude"
+			action
+				"unfettered headpat attitude" --
+			label nochange
+			choice
+				`	"It's just four words."`
+				`	"Nevermind, just give me the credits."`
+					goto money
+			`	As soon as you finish speaking, one of the Unfettered runs up and stabs you with a machete.`
```
Comment (Hurleveur) [inline data/side missions.txt]:
```suggestion
			`	As soon as you finish speaking, one of the Unfettered runs through you with a machete and shouts "I'm not cute!"`
```
Mb like this to add a ref to the spaceport msg haha?

**Review by Hurleveur — COMMENTED** [review]
Looks good, I'll merge soon

File: data/side missions.txt (hunk 4 above)
Comment (TheGiraffe3) [reply data/side missions.txt]: Wouldn't it be "you through"?
```suggestion
			`	As soon as you finish speaking, one of the Unfettered runs you through with a machete and shouts "I'm not cute!"`
```

**Review by Hurleveur — APPROVED** [review]

**Pull request merged**